packages/git/src/browser/blame/blame-decorator.ts

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { inject, injectable } from 'inversify';
+import {
+    EditorManager, EditorDecorationsService, TextEditor, EditorDecoration, EditorDecorationOptions, Range, Position, DecorationStyle, DecorationStyles
+} from '@theia/editor/lib/browser';
+import { GitFileBlame, Commit } from '../../common';
+import { Disposable, DisposableCollection } from '@theia/core';
+
+const maxWidth = 50; // character
```

Comment:
please put global constants and `AppliedDecorations` under `BlameDecorator` namespace

Reply:
done.